src/geo/map.js

```diff
@@ -19,10 +19,9 @@ var Map = Model.extend({
     drag: true,
     keyboard: true,
     provider: 'leaflet',
-    // enforce client-side rendering using GeoJSON vector tiles
-    vector: false,
     popupsEnabled: true,
-    isFeatureInteractivityEnabled: false
+    isFeatureInteractivityEnabled: false,
+    featureCount: 0
```

Comment:
Let's change this to `Infinity` before releasing this so that things continue working as before in case the Maps API doesn't return stats. (cc: @donflopez)